department-of-veterans-affairs/caseflow
# Queue | Fix displaying assign vs review judge task views

### Description
In #6819, I replaced instances of `taskType` with `action`, and normalized assign/review action keys to lower-case.

I failed to realize that in the case of `QueueApp.routedJudgeQueueList`, the `taskType`/`action` parameter was passed from the `<PageRoute/>` declaration, rather than coming from the task / server, leading to judges being unable to view the Assign cases page. Currently, judge assignment pages / flows lack any automated tests, which would (theoretically) have helped catch this.

### Acceptance Criteria
- [ ] Judge Assign Task list view loads correctly on click Assign X Tasks

### Testing Plan
1. Go to `/queue` as `BVAAABSHIRE`, click Assign Tasks
2. Confirm assign tasks page appears
